bin/pana.dart

```diff
@@ -44,8 +44,8 @@ void _printHelp({String errorMessage}) {
     print(red.wrap(errorMessage));
     print('');
   }
-  print('''Usage: pana [<options>] <package> [<version>]
-       pana [<options>] --source path <directory>
+  print('''pana [<options>] --source path <absolute directory path>
```

Comment:
Can we implement relative paths, e.g. by normalizing them if needed?

Reply:
Agreed: https://github.com/dart-lang/pana/pull/392